nrkno/sofie-hyperdeck-connection
# fix: don't return videoFormat: null for async transport updates

## About the Contributor
One of the maintainers of the Hyperdeck Companion module.


## Type of Contribution

This is a: 
Bug fix


## Current Behavior
The async transportStatus update will send a value of null for videoFormat if no other value is present. This is incorrect, because the Hyperdeck only sends parameters that have changed since the last transport status update, so if there is no videoFormat parameter present, one should not be passed upstream either.


## New Behavior
Remove the fallback null value for videoFormat


## Testing Instructions
Start and stop playback, and verify that there is no videoFormat parameter being passed to notify.transport handlers.


## Other Information



## Status

- [✅ ] PR is ready to be reviewed.
- [✅ ] The functionality has been tested by the author.
